botany_importer.py

```diff
@@ -92,12 +89,10 @@ def process_barcode(self, barcode, filepath_list):
 
     def build_filename_map(self, full_path):
         full_path = full_path.lower()
-
         if not self.check_for_valid_image(full_path):
             return
-
         filename = os.path.basename(full_path)
-        matched = re.match(botany_importer_config.BOTANY_REGEX, filename.lower())
+        matched = re.match(self.botany_importer_config['BOTANY_REGEX'], filename.lower())
```

Comment:
testing this line out before pushing, and it still works fine in using the regex expressions from json, without explicitly defining them as raw strings `r`. let me know if you think this could still lead to problems.